packages/common/src/core/schemas/internal/discussions/EntityUiSchemaDiscussionsSettings.ts

```diff
@@ -50,6 +51,67 @@ export const buildUiSchema = (
     context.hubLicense === "hub-premium" &&
     options.type !== "Hub Site Application"
   ) {
+    const orgFacets = {
+      label:
+        "{{shared.fields.allowedChannelIds.facets.organization.label:translate}}",
+      key: "organization",
+      display: "single-select",
+      operation: "OR",
+      options: [
+        {
+          label:
+            "{{shared.fields.allowedChannelIds.facets.organization.options.organization.label:translate}}",
+          key: "organization",
+          predicates: [
+            {
+              orgId: [context.currentUser.orgId],
+            },
+          ],
+        },
+        {
+          label:
+            "{{shared.fields.allowedChannelIds.facets.organization.options.world.label:translate}}",
+          key: "world",
+          predicates: [
+            {
+              orgId: [null],
+            },
+          ],
+        },
+      ],
+    };
+
+    if (context.currentUser.cOrgId) {
```

Comment:
Is this right? When I auth as `juliana_pa` go to the asheville site and use `ember inspector => Routes` to drill into `appSettings.context`, I notice that `context.communityOrgId` has a value of `uFEMdY4VMonzH8sG` while `context.currentUser.cOrgId` is `undefined` (key is absent altogether).